seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-2/src/test/resources/jdbc_trino_source_and_console .conf

```diff
@@ -0,0 +1,48 @@
+#
```

Comment:
I can't find the place to trigger this test case.

Reply:
remove unued file